package.json

```diff
@@ -43,7 +43,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js icons"
   },
-  "version": "2.35.0",
+  "version": "2.35.1",
```

Comment:
Oh, this just reminded me that I didn't release 2.35 yet because I got focused on discourse stuff :)

But on the 'version' note.
This is technically a new 'feature' - as we are introducing new class name, so this will be 2.36 and we need to add this to "What's new" table as well.